gui

```diff
@@ -435,7 +435,14 @@ runonce <<"EOF"
   #rename 'KeePassX' to 'KeePassXC'
   rm -rf "${DIRECTORY}/apps/KeePassX"
   mv -f "${DIRECTORY}/data/status/KeePassX" "${DIRECTORY}/data/status/KeePassXC" 2>/dev/null
-  
+  true
+EOF
+
+#rename TeamViewer Host to TeamViewer
+runonce <<"EOF"
+  #rename current app 'TeamViewer Host' to 'TeamViewer' if not already present
+  [ ! -d "${DIRECTORY}/apps/TeamViewer" ] && mv -f "${DIRECTORY}/apps/TeamViewer Host" "${DIRECTORY}/apps/TeamViewer" && sed -i 's;TeamViewer Host|Internet;TeamViewer|Internet;g' "${DIRECTORY}/etc/categories"
+  mv -f "${DIRECTORY}/data/status/TeamViewer Host" "${DIRECTORY}/data/status/TeamViewer" 2>/dev/null
   true
 EOF
```

Comment:
Three problems with this line:

1. If the user updates everything, the `${DIRECTORY}/apps/TeamViewer` directory will be created at the same time the `gui` script is updated with this code. As a result, this code will never be run, as it's checking for the `TeamViewer` app to *not* exist.
2. The `etc/categories` file should not be edited for any reason.
3. This whole runonce does not need to be in its own separate location. Please put this app-renaming stuff in the pre-existing runonce and follow that format.

Reply:
I was not aware that the app would be re-installed if the update folder contained an app folder that does not exist on the local install and the status file for that app existed.

I tested locally and that does seem to be the case, that explains your method which I thought did not work.

Reply:
actually I guess yad now calls the GUI any time the user clicks a button. which is good in this case which means that new runonce files get run without the user re-loading the GUI itself

Reply:
@Botspot seehttps://github.com/Botspot/pi-apps/pull/1994/commits/92ae9bd9e229a03521485685d92c66d622f5800c for the proper method which triggers the updater after doing the initial update

Reply:
> actually I guess yad now calls the GUI any time the user clicks a button. which is good in this case which means that new runonce files get run without the user re-loading the GUI itself

actually unfortunately the yad called processes never reach the runonce (they exit before any runonce is called) so this does still require a re-launch of the gui

Reply:
alright... the question now is. do we hack the runonce into `set-status` like we did with the new-gui update and make this official practice when renaming new apps? because that is the only part of any script that is called after running an update